vignettes/get_data_inboveg.Rmd

```diff
@@ -125,34 +148,37 @@ all_header_info <- get_inboveg_header(con)
 
 
 ## Recording information
-The function  `get_inboveg_recordings` queries the INBOVEG database for relevé information (which species were recorded in which plots and in which vegetation layers with which cover) for one or more surveys.
+The function  `get_inboveg_recording` queries the INBOVEG database for relevé information (which species were recorded in which plots and in which vegetation layers with which cover) for one or more surveys.
```

Comment:
Add brackets to make it look like a function. Then pkgdown can detect it and automatically create a link to the helppage.
```suggestion
The function  `get_inboveg_recording()` queries the INBOVEG database for relevé information (which species were recorded in which plots and in which vegetation layers with which cover) for one or more surveys.
```